### PR TITLE
fix: unable to resolve index.tsx dependencies

### PR DESCRIPTION
**Pull request by SyMind**
https://github.com/dependents/node-dependency-tree/issues/118

### CODE REVIEW

**Comment by kodai3** [comment]
It took quite a while to find out this RP fix my issue of madge keep skipping some of my dependencies.
Thanks and hope it will be merged soon

**Comment by PabloLION** [comment]
Sorry for the inactivity. I'll try to review and merge it today.

**Comment by StefanTobler** [comment]
Bumping this!

File: test/typescript.js [hunk 1]
```diff
@@ -2,7 +2,6 @@
 'use strict';
 
 const madge = require('../lib/api');
-const ts = require('typescript');
```
[the remark this answers is not in the document]
Comment (SyMind) [reply test/typescript.js:5]: done.

**Comment by odinho** [comment]
Ah, this would probably fix my issue, so won't open up yet another bug on that.

Some new syntax in TS 'using' that the older parsers don't support.

Thanks for doing this! :)

**Comment by basicButtons** [comment]
> @SyMind Some of these changes are also included in #379 . Does it change anything wrt to the issue you're experiencing and targeting here?其中一些更改也包含在 #379 中。它是否改变了您在这里遇到和针对的问题的任何内容？

It's not resolved.

**Pull request closed without merging**